### ISSUE TITLE
Time to deprecate this repo?

### ISSUE THREAD

**Issue by mkborregaard** [JuliaStats/Roadmap.jl#23]
It doesn't seem to be used for any of the roadmap discussion actively taking place.

**Comment by andreasnoack** [comment]
Yes. It's hasn't really been used for a long time so it might be a good idea to deprecate this. Either by updating the README or moving it to JuliaArchive.

**Comment by datnamer** [comment]
@mkborregaard  where is the discussion taking place?

**Comment by mkborregaard** [comment]
e.g. here https://github.com/JuliaStats/Stats.jl/pull/4

**Comment by ararslan** [comment]
We can just use GitHub's archive functionality.

**Comment by matbesancon** [comment]
So are we archiving it? Doing it using Github archiving in ~12 hours if no objection. cc @ararslan
